### PR TITLE
feat: customize capabilities from file

### PR DESCRIPTION
**Pull request by link89**
Enable to set capabilities from file.
This is useful when you have a lot of arguments to tune.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -20,13 +22,21 @@ export default {
...
         const browserNameString = browserName.match(/([^@:]+)/);
+        const browserFormalName = browserNameString[1].trim().toLocaleLowerCase();
         let version = browserName.match(/@([^:]+)/);
         let platform = browserName.match(/:(.+)/);
 
         version = version ? version[1] : undefined; // eslint-disable-line no-undefined
         platform = platform ? platform[1] : undefined; // eslint-disable-line no-undefined
-        const browser = await new Builder().forBrowser(browserNameString[1], version, platform).usingServer(this.seleniumServer).build();
 
+        const builder = new Builder().forBrowser(browserFormalName, version, platform).usingServer(this.seleniumServer);
```
Comment (alexschwantes) [inline src/index.js]: Why did you remove the await here?

Comment (link89) [reply src/index.js]: Here we seperate builder from build in order to add more options latter.
And here is when browser actually opened:
await builder.build()

**Comment by ondrejbartas** [comment]
👍

**Comment by link89** [comment]
@alexschwantes Hi, it seems like after push - f, your requested changes is invalid.

**Comment by ondrejbartas** [comment]
@link89 @alexschwantes Can you make this work? I really need this for my project.

Thanks for you work

**Comment by link89** [comment]
@ondrejbartas You can fetch my feature branch and build a local version as a temporary solution.

File: src/index.js [hunk 2]
```diff
@@ -20,13 +36,24 @@ export default {
...
+        const optionSetter = optionSetterMaps[browserFormalName];
+
+        if (optionSetter && existsSync(this.capabilities)) {
+            const caps = JSON.parse(readFileSync(this.capabilities))[browserName.trim()];
```
Comment (alexschwantes) [inline src/index.js]: From line 26, the capabilities can be in a file or in `process.env.SELENIUM_CAPABILITIES`. If it is in an environment varialbe will this work? It should also be mentioned in the readme that it is possible to sent it in an env var.
The default for this should be no custom capabilities set.

Comment (link89) [reply src/index.js]: SELENIUM_CAPABILITIES is used to override the default path of capabilities file. By default is capabilities.json. I just provide this env-var in case some one may need to use this (like concurrent job using different configuration)

Comment (alexschwantes) [reply src/index.js]: Cool, somehow I wrongly read that it could also be passed as text into the env var.
